VaticanBAV/et/BAVet141.xml

```diff
@@ -284,7 +284,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                       
                     <decoDesc>
                       <decoNote type="ornamentation" xml:id="d1">
-                        <desc>The Psalms are divided into ten lines, which are marked alternately with red and black dots.</desc>
+                        <desc>The catalogue statement ‘Psalmi deni lineis dividuntur,
+                          quae constant punctis alterna vice rubris et nigris.’ probably indicates that 
```

Comment:
Can this be marked as Latin text?

Reply:
I decided against it, because this would them make this manuscript turn up in any searches filtering for Latin text in manuscripts, even when the manuscript itself does not contain any Latin...not ideal